File: curso-python/ex026.py
#Faça um programa que leia uma frase pelo teclado e mostre
# quantas vezes aparece a letra A
# em que posição ela aparece a primeira vez
# em que posição ela aparece a última vez
frase = input("digite uma frase qualquer: ")
print(f"quantas vezes aparece a letra A?: {frase.count('a')}")
print(f"em que posição o primeiro A apareceu?: {frase.find('a')} ")
print(f"em que posição o A aparece a última vez?: {frase.rfind("a")}")

File: javascript-exercicios/ex026.js
const prompt = require("prompt-sync")();

let frase = prompt("digite uma frase qualquer: ")
frase = frase.toLowerCase(); 

const letra = 'a';
const qtd = frase.split(letra).length - 1; // conta quantas vezes 'a' aparece
const primeira = frase.indexOf(letra);     // primeira ocorrência
const ultima = frase.lastIndexOf(letra);   // última ocorrência

console.log(`Quantas vezes aparece a letra 'a'?: ${qtd}`);
console.log(`Em que posição o primeiro 'a' apareceu?: ${primeira}`);
console.log(`Em que posição o 'a' aparece a última vez?: ${ultima}`);
